localstack/ext/java/src/main/java/cloud/localstack/Localstack.java

```diff
@@ -0,0 +1,297 @@
+package cloud.localstack;
+
+import com.amazonaws.util.IOUtils;
+import org.ow2.proactive.process_tree_killer.ProcessTree;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.logging.Logger;
+
+/**
+ * Simple JUnit wrapper that automatically downloads, installs, starts,
```

Comment:
Nitpick: Can we update the comment - technically there is nothing JUnit specific in this class anymore.